pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -0,0 +1,56 @@
+But in situations where master is used in code and previously was part of a master/slave terminology pairing, we commit to changing the language we use by DATEDATEDATE **TKTKTKTK commit to a timeline to change the main branch?? TKTKTKTKTK**.
+
+But changing language is not enough.
+
+## Diversifying Recruiting
+
+In our discussions, we quickly noted that the Mixxx contributor base is largely white, male, and either American or European.
+For some of our coders, the experience of working on Mixxx has been a doorway to employment at some of the most prestigious software companies, and we want that door to be open to anyone.
+It is our goal to make the Mixxx project as inclusive, accessible and welcoming as possible.
+As such, it is our responsibility to actively seek out and promote the participation of traditionally underserved groups.
+
+As a small volunteer-led project with no regular income, budget, or employees, we don't yet know for sure what the demographic makeup of our contributors is -- but we are aware of the research that shows that blind admission and hiring tends to result in less diverse outcomes.
+To that end we plan to sponsor an [Outreachy](https://www.outreachy.org/) intern in December 2020, the next available cycle.
+Outreachy’s stated goal is to provide "internships to work in open source and free software, for women (both cis and trans), trans men, and genderqueer people" and "residents and nationals of the United States of any gender who are Black/African American, Hispanic/Latin@, Native American/American Indian, Alaska Native, Native Hawaiian, or Pacific Islander."
+They encourage "anyone who faces under-representation, systemic bias, or discrimination in the technology industry of their country" to apply.
+
+Our goal is that this Outreachy intern will be the first of an increasing number of valued contributors from underserved communities that will help us be a healthier, more sustainable open source project.
+With your help, we intend to continue the internship process semiannually for at least 5 cycles and at that time assess if we should continue with the program or if our diversification efforts would be better served with a new approach.
+
+An Outreachy intern will cost Mixxx $6,500 to sponsor.
+Because we don't have the money to do this ourselves, we are asking the Mixxx community to step up and pitch in to help raise the necessary funds for this sponsorship.
+We have never and will never charge for Mixxx, so this is a chance for you to contribute money that will go directly to a person working on the project.
+
+`DETAILS ABOUT CAMPAIGN, LINK`
+
+We hope by taking this stand for diversity and inclusion our medium-sized open source project can show that doing the right thing is not just about renaming branches, but doing the hard work of changing culture, even on a small scale.
+
```

Comment:
I'd like to add another paragraph that says something along the lines of:

we're aware that replacing some terms and gparticipating in outreachy might seem little considering the discrimination that POC and black people in particular face every day (and being at risk of even getting killed by the police). As a project that depends on donations and volunteer work, we are not sure what else we can do. We'd invite the community to get in touch with us on Zulip and discuss further measures that we can take to make our community more welcoming and to take a stand against racism and discrimination. Keep protesting!

Reply:
might seem a little what? :)

Reply:
Sorry, I meant something like "it's not much compared to the problems"

Reply:
I would skip the "we are not sure what else we can do" phrasing as it borders on sounding like a request for marginalized communities to do the labor of telling Mixxx what to do.

How about
"We're aware that replacing some terms and participating in Outreachy might seem insignificant in light of the discrimination and violence that people of color and Black people in particular face every day. As a project that depends on donations and volunteer work, we invite all our collaborators and users to discuss further community action on Zulip as we continue to stand against racism and discrimination. Keep protesting!"

Reply:
Great, my suggestion wasn't supposed to be worded properly, I tried to get the meaning across ;-)

Reply:
fixed